src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
don't need this enum

Reply:
Should I simplify code to be comparing Strings, or even random integers as conditions of particular figure creation?